### PR TITLE
migrate api-server to ts

### PR DESCRIPTION
**Pull request by obadrawi**

### CODE REVIEW

File: api-server/src/server/app.ts [hunk 1]
```diff
@@ -64,12 +64,10 @@ function init(varkesConfigPath = null, currentPath = "", nodePortParam = null) {
     app.get("/console", function (req, res) {
         res.sendFile(path.resolve(__dirname, "resources/console.html"))
     })
-
-    return new Promise(function (resolve, reject) {
-        resolve(app)
-    })
+    return app;
+    // return new Promise(function (resolve, reject) {
```
Comment (a-thaler) [inline api-server/src/server/app.ts]: please remove commented lines

File: api-server/package.json [hunk 2]
```diff
@@ -30,11 +32,23 @@
     "request": "^2.88.0",
     "request-promise": "^4.2.4",
     "swagger-ui-dist": "^3.22.1",
-    "winston": "^3.2.1"
+    "copyfiles": "^2.1.0",
```
Comment (a-thaler) [inline api-server/package.json]: copyfiles should be a devDependency I guess

File: api-server/src/server/app.ts (hunk 1 above)
Comment (obadrawi) [reply api-server/src/server/app.ts]: Done

File: api-server/package.json (hunk 2 above)
Comment (obadrawi) [reply api-server/package.json]: Done

**Pull request merged**